bucket/flutter.json

```
@@ -1,14 +1,14 @@
 {
     "homepage": "https://flutter.dev/",
-    "version": "1.5.4-hotfix.2",
+    "version": "1.12.13-hotfix.5",
     "license": "BSD-3-Clause",
     "url": [
-        "https://storage.googleapis.com/flutter_infra/releases/stable/windows/flutter_windows_v1.5.4-hotfix.2-stable.zip",
+        "https://storage.googleapis.com/flutter_infra/releases/stable/windows/flutter_windows_v1.12.13+hotfix.5-stable.zip",
         "https://raw.githubusercontent.com/lukesampson/scoop-extras/master/scripts/flutter-dev-setup.ps1"
     ],
     "extract_dir": "flutter",
     "hash": [
-        "1d5e1779430be6bbb4fa49f4bccc7c5d24a76f3d2a100129e09b4b8eef33b1b4",
+        "F86E162C3C732D4FC85CE85B4EB9D4E0EEAF3F6020F7FD44DA247BFB87C7AC40",
```

We usually keep hashes lower-cased (hash checker is case-agonstic).

```suggestion
        "f86e162c3c732d4fc85ce85b4eb9d4e0eeaf3f6020f7fd44da247bfb87c7ac40",
```